### DISCUSSION TITLE
Issue with Folder Renaming and Sync

### DISCUSSION THREAD

**Discussion by wilmaldonado** [iterate-ch/cyberduck#15438] [Q&A]
**Folder Renaming Changes and Sync**
•	Problem: Renaming a folder in MD sometimes causes synchronization issues, resulting in two folders with different names and split files.

**Comment by tatze96** [comment]
- Which version of Mountain Duck is in use?
- Which protocol is in use?

If the issue still persists, send us the logging output with enabled [debug logging](https://docs.cyberduck.io/mountainduck/support/#debug-log) right after the issue occurs again to support@mountainduck.io with the ticket reference GH-15438 for further investigation.

**Comment by wilmaldonado** [comment]
Were you able to look through the logs that I had previously provided? I had included some additional logs as requested.

> https://web.tresorit.com/l/VuUoe#ADuhhbEPRO3JGIMgrGRpRA
